.github/workflows/kedro-datasets.yml

```diff
@@ -58,9 +58,19 @@ jobs:
         run: |
           cd kedro-datasets
           uv pip install --system "kedro-datasets[docs,test,experimental] @ ."
-      - name: Documentation check for kedro-datasets
+      - name: Build datasets documentation
         run: |
-          make check-datasets-docs
+          make build-datasets-docs
+      - name: Check markdown formatting
+        run: |
+          make fix-markdownlint
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v2
+        with:
+          fail: false
```

Comment:
I assume this check will pass after the release?

Reply:
Yes when everything is in place I will remove this line and make sure all links between docs are working.